lib/exabgp/bgp/message/update/attribute/mprnlri.py

```diff
@@ -171,6 +174,10 @@ def unpack (cls, data, negotiated):
 				length,nlri = NLRI.unpack(afi,safi,data,addpath,'',IN.ANNOUNCED)
 				nlris.append(nlri)
 
+			if length == 0:
```

Comment:
If the subclass is buggy the code should be fixed in the buggy sub-class not the caller. Can I remove this code please ?